app/src/main/java/app/foodfinderapp/dto/Food.kt

```diff
@@ -1,6 +1,8 @@
 package app.foodfinderapp.dto
 
-data class Food(var foodID:Int, var foodName: String, var foodPrice: Float, var foodCategory: String) {
+import com.google.gson.annotations.SerializedName
+
+data class Food(var foodID:Int, @SerializedName("foodName")var foodName: String, var foodPrice: Float,@SerializedName("foodCategory") var foodCategory: String) {
```

Comment:
@SerializedName is only required if the JSON field names do not match the data class attribute names.
In a case like this, where the names match, @SerializedName is not providing value... and thus, is extra code that is not needed.